From 7dcf80ba834e54d6745575086eb4e4c4db1cad92 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sat, 16 Feb 2008 16:09:51 -0500
Subject: [PATCH] do not use the '-s' flag with `install`

The `install` program strips binaries when given the '-s' flag.  This step
should be left up to package maintainers to handle the stripping, especially
since the `install` program will always execute `strip` -- this is no good
for cross-compiling for example.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 progs/Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/progs/Makefile b/progs/Makefile
index 9d25a78..d711656 100644
--- a/progs/Makefile
+++ b/progs/Makefile
@@ -17,7 +17,7 @@ $(PROGS): %: %.o
 install: all
 	mkdir -p -m 0755 $(SBINDIR)
 	for p in $(PROGS) ; do \
-		install -s -m 0755 $$p $(SBINDIR) ; \
+		install -m 0755 $$p $(SBINDIR) ; \
 	done
 
 clean:
-- 
1.5.4

